vite.config.ts

```diff
@@ -0,0 +1,52 @@
+import { NodeGlobalsPolyfillPlugin } from '@esbuild-plugins/node-globals-polyfill'
+import { NodeModulesPolyfillPlugin } from '@esbuild-plugins/node-modules-polyfill'
+import react from '@vitejs/plugin-react'
+import path from 'path'
+import { defineConfig } from 'vite'
+import svgr from 'vite-plugin-svgr'
+
+export default defineConfig({
```

Comment:
This config lets us run the deprecated demo app. Was useful to get Vite working with a basic demo of our library.

Likely will be worth keeping around. Eventually, Vite will likely use esbuild to create a bundle, at which point we can probably combine this config with the esbuild config.